### ISSUE TITLE
Positions when max value is 6

### ISSUE THREAD

**Issue by ViorelEremia** [IonDen/ion.rangeSlider#559]
Hi,

We use ion range slider as an option type in an open source wordpress plugin unyson: https://wordpress.org/plugins/unyson/
Some developers using this option type have noticed an issue when the max value is set to 6. 
To be better understood I created a demo here: http://jsfiddle.net/72pcotsu/13/

Thank you for the great plugin.

**Comment by IonDen** [comment]
Hi, you mean that grid is showing wrong numbers? Yes it could happen with small scales. To overcome this problem, please use `grid_snap: true` param, like this: http://jsfiddle.net/72pcotsu/13/

**Comment by ViorelEremia** [comment]
Yes, with ```grid_snap: true``` for the unyson plugin is fine is working.

> Hi, you mean that grid is showing wrong numbers?

![ion rangeslider test playground jsfiddle](https://user-images.githubusercontent.com/28967378/42206196-16aaa2de-7eaf-11e8-96a3-a5d32e73ac28.png)
